### ISSUE TITLE
How to access analog pin ? 

### ISSUE THREAD

**Issue by aessig** [shokai/node-arduino-firmata#16]
I tried to access analog pin with your library without success. Using "A0" as pin name, using the pin number (14 for A0 on Leonardo). I even tried to use analogWrite with only the analog pin value (0,1,2,3...) and the pin number (14), but nothing seems to work. I'm using StandardFirmata. Can you help me ?

**Comment by shokai** [comment]
yes, firmata does not support analog input pin to use digital-output.

`IS_PIN_PWM` macro is defined in `Arduino.app/Contents/Java/hardware/arduino/avr/cores/arduino/Arduino.h`
```C
// Leonardo
#elif defined(__AVR_ATmega32U4__)
#define TOTAL_ANALOG_PINS       12
#define TOTAL_PINS              30 // 14 digital + 12 analog + 4 SPI (D14-D17 on ISP header)
#define VERSION_BLINK_PIN       13
#define IS_PIN_DIGITAL(p)       ((p) >= 0 && (p) < TOTAL_PINS)
#define IS_PIN_ANALOG(p)        ((p) >= 18 && (p) < TOTAL_PINS)
#define IS_PIN_PWM(p)           ((p) == 3 || (p) == 5 || (p) == 6 || (p) == 9 || (p) == 10 || (p) == 11 || (p) == 13)
```

So we cannot use pin14 as digital output.

Probably if you modify `setPinModeCallback` function in StandardFirmata, it will be able to..

**Comment by aessig** [comment]
Thanks. I’ll try to modify setPinModeCallback. Cheers
